### ISSUE TITLE
WA - Incorrect data in OBX-17

### ISSUE THREAD

**Issue by Adrian-Brewster** [CDCgov/prime-reportstream#2347]
The below email came in from Washington on 9/23

I’m reporting an issue where we are receiving the wrong information in OBX 17. We did observe this when we first started validating the data: 8/13/21: CDC device ID used in test method field for about 75% of records. Xpress test method used for the the remaining 25%  However, because we get the same information from all the other submitters we see coming through AIMS, my team member didn’t follow up on this one.  We would expect to see the name of the text string of the test method in OBX 17.2 or 17.4.  Is this a ‘bug’ on the USDS side or were you instructed to format that way?  If it’s a bug, would you please enter a ticket for it?

**Comment by RickHawesUSDS** [comment]
I agree that this is a bug. We should put the text name in OBX-17.2.
